hummingbot/market/bitfinex/__init__.py

```diff
@@ -1,11 +1,22 @@
 import re
+from decimal import Decimal
 
 BITFINEX_REST_URL = "https://api-pub.bitfinex.com/v2"
 BITFINEX_REST_AUTH_URL = "https://api.bitfinex.com/v2"
 BITFINEX_WS_URI = "wss://api-pub.bitfinex.com/ws/2"
 BITFINEX_WS_AUTH_URI = "wss://api.bitfinex.com/ws/2"
 
-TRADING_PAIR_SPLITTER = re.compile(r"^(\w+)(BTC|ETH|BNB|XRP|USD|USDT|USDC|USDS|TUSD|PAX|TRX|BUSD|NGN)$")
+TRADING_PAIR_SPLITTER = re.compile(r"^(\w+)(BTC|ETH|BNB|XRP|USD|USDT|USDC|USDS|TUSD|PAX|TRX|BUSD|NGN|LTC|ETC|XMR|DASH)$")
+
+
+MIN_BASE_AMOUNT_INCREMENT = Decimal("0.01")
+
+# this values ​​set by empirically way, because the bitfinex-market does not have
+# these values. maybe later it will be in market-api.
+BITFINEX_QUOTE_INCREMENT = 0.01
+BITFINEX_BASE_INCREMENT = 1e-8
+TAKER_FEE = Decimal("0.002")
+MAKER_FEE = Decimal("0.001")
```

Comment:
Do keep updated on any change on the API. This might cause some issues especially if BitFinex decides to change their Trading Fee policy.